alot/settings/manager.py

```diff
@@ -433,7 +433,7 @@ def get_keybindings(self, mode):
                 if value and value != '':
                     globalmaps[key] = value
         # get rid of empty commands left in mode bindings
-        for k, v in modemaps.items():
+        for k, v in list(modemaps.items()):
```

Comment:
Why is this list cast needed?

Reply:
It's possible that it was autogenerated by 2to3, I'll have to look at it.

Reply:
I'm looking at this more, since the below code modifies modemaps it might be necessary to use list here to avoid the iterator of the view dying horribly.